Frends.Csv.Tests/Tests.cs

```diff
@@ -328,7 +328,7 @@ public void ParseAndWriteShouldUseSeparateCultures()
                 },
```

Comment:
https://github.com/FrendsPlatform/Frends.Csv/blob/157ff7db06a10ba1391ac05508fb2c7e5c9ea12a/Frends.Csv.Tests/Tests.cs#L158
Please use `IsNull` instead of `AreEqual`.

https://github.com/FrendsPlatform/Frends.Csv/blob/157ff7db06a10ba1391ac05508fb2c7e5c9ea12a/Frends.Csv.Tests/Tests.cs#L166
Same here.

https://github.com/FrendsPlatform/Frends.Csv/blob/157ff7db06a10ba1391ac05508fb2c7e5c9ea12a/Frends.Csv.Tests/Tests.cs#L195
Please use `IsFalse` instead of `AreEqual`.